src/views/CoCurricularTranscript/guestTranscript.tsx

```diff
@@ -0,0 +1,116 @@
+import { MembershipHistory, MembershipHistorySession } from 'services/user';
+import { StudentProfileInfo } from 'services/user';
+import { Class } from 'services/peopleSearch';
+import { Participation } from 'services/membership';
+import { TranscriptItems } from 'services/transcript';
+
+const participation1: Participation = Participation.Member;
+const currentYear: number = new Date().getFullYear();
+const session1: MembershipHistorySession = {
+  MembershipID: -1,
+  SessionCode: currentYear - 1 + '09',
+  Participation: participation1,
+};
+const activity1: MembershipHistory = {
+  ActivityCode: 'ISO',
+  ActivityDescription: 'International Student Organization',
+  ActivityImagePath: '',
+  Sessions: [session1],
+  LatestDate: currentYear - 1 + '-12-15T00:00:00',
+};
+const participation2: Participation = Participation.Member;
+const session2: MembershipHistorySession = {
+  MembershipID: -1,
+  SessionCode: currentYear + '01',
+  Participation: participation2,
+};
+const activity2: MembershipHistory = {
+  ActivityCode: '360',
+  ActivityDescription: '360.gordon.edu',
+  ActivityImagePath: '',
+  Sessions: [session2],
+  LatestDate: currentYear + '-05-10T00:00:00',
+};
+export const transcriptItems1: TranscriptItems = {
+  honors: [],
+  experiences: [],
+  service: [],
+  activities: [activity1, activity2],
+};
+export const student1: StudentProfileInfo = {
```

Comment:
I think we could rename this for more clarity, maybe
```suggestion
export const exampleStudentProfile: StudentProfileInfo = {
```